src/Service/Geocoder.php

```diff
@@ -77,6 +81,18 @@ private function getGeocoder()
         return $this->geocoder;
     }
 
+    private function createGeocodeEarthProvider() {
+        $rateLimiter = RateLimiterMiddleware::perSecond($this->rateLimitPerSecond, $this->rateLimiterStore);
```

Comment:
The `$limit` should be 10 requests per second for Geocode Earth

https://geocode.earth/#pricing

Reply:
I think there was an other problem is that I used the same rate limit store as the open cage one. see here : ab8dbda70a0d0ef1ae5edf398b53255c9d7d38eb